pages/common/pg_restore.md

```diff
@@ -0,0 +1,23 @@
+# pg_restore
+
+> Restore a PostgreSQL database from an archive file created by pg_dump
+
+- restore an archive into an existed database
+
+`pg_restore -d {{db_name}} {{archive_file.dump}}`
+
+- customize username
+
+`pg_restore -U {{username}} -d {{db_name}} {{archive_file.dump}}`
+
+- customize host and port
+
+`pg_restore -h {{host}} -p {{port}} -d {{db_name}} {{archive_file.dump}}`
+
+- clean database object before creating them
```

Comment:
Updated the above. For the "them" meaning, the "database object" should be plural, and the "them" means the "database objects".